### PR TITLE
[jsk_perception] Add dynamic_reconfigure feature to ImagePublisher

### PR DESCRIPTION
**Pull request by wkentaro**

### CODE REVIEW

**Comment by garaemon** [comment]
Use dynamic_reconfigure

File: jsk_perception/scripts/image_publisher.py [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+    config['file_name'] = os.path.abspath(file_name)
+    return config
+
+
 rospy.init_node("image_publisher")
 
+dynamic_reconfigure.server.Server(ImagePublisherConfig, _cb_dyn_reconfig)
+file_name = rospy.get_param("~file_name")
+file_name = os.path.abspath(file_name)
+rospy.set_param("~file_name", file_name)
```
Comment (garaemon) [inline jsk_perception/scripts/image_publisher.py]: You don't need this line

**Comment by wkentaro** [comment]
I updated. Please check it.

File: jsk_perception/scripts/image_publisher.py [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+    global file_name
+    file_name = config['file_name']
+    config['file_name'] = os.path.abspath(file_name)
+    return config
+
+
 rospy.init_node("image_publisher")
 
+dynamic_reconfigure.server.Server(ImagePublisherConfig, _cb_dyn_reconfig)
+file_name = rospy.get_param("~file_name")
```
Comment (garaemon) [inline jsk_perception/scripts/image_publisher.py]: dynamic_reconfigure should read ~file_name parameter

Comment (wkentaro) [inline jsk_perception/scripts/image_publisher.py]: What exactly should I change?

Comment (garaemon) [inline jsk_perception/scripts/image_publisher.py]: Just remove the line of `file_name = ...`

File: jsk_perception/scripts/image_publisher.py [hunk 3]
```diff
@@ -1,16 +1,28 @@
...
 import cv_bridge
+import dynamic_reconfigure.server
+
+from jsk_perception.cfg import ImagePublisherConfig
 from sensor_msgs.msg import Image, CameraInfo
 
 
+def _cb_dyn_reconfig(config, level):
+    global file_name
+    file_name = config['file_name']
```
Comment (garaemon) [inline jsk_perception/scripts/image_publisher.py]: ~file_name is read here

File: jsk_perception/scripts/image_publisher.py (hunk 2 above)
Comment (garaemon) [inline jsk_perception/scripts/image_publisher.py]: I will merge this pull request, please send another pull request to fix this.

**Comment by wkentaro** [comment]
sorry for late.
I will send another PR ASAP.

**Pull request merged**